external_services/models.py

```diff
@@ -166,6 +166,48 @@ def get_url(self, replace=None, kwargs={}): # pylint: disable=dangerous-default-
         return reverse('lti-login', kwargs=kwargs)
 
 
+class LTI1p3Service(LinkService):
+    '''
+    Settings for a remote LTI v1.3 Tool.
+    '''
+    # Verbose names are intentionally non-translated, because they would not make sense in Finnish.
+    login_url = models.CharField(
+        verbose_name="Login initiation URL",
+        max_length=256,
+    )
+    client_id = models.CharField(
```

Comment:
client_id should. Not sure about deployment_id, we do not use it yet for much anything, but it is included because specification wants it, and maybe in future there will be some use for multiple deployments.

Reply:
So let's add the uniqueness constraint to the `client_id`. I found this Q&A, and I'm not sure what to make out of it since I'm not that familiar with the terminology.

https://www.imsglobal.org/forum/learning-tools-interoperability-public-forum/197896

> 1) For a given issuer, there can be multiple client_id values. Can the service URLs vary based on the client_id, or are they always linked only to the issuer?
> 
> Yes there can be multiple client_ids for an issuer pointing to the same tool. The way that you would make a distinction between them is by having an id in the in the OIDC login initiation url as a path or query parameter.